### PR TITLE
Update Google Ads.json

### PR DESCRIPTION
**Pull request by AaronjSugarman**

### CODE REVIEW

File: services/Google Ads.json [hunk 1]
```diff
@@ -35,6 +35,10 @@
       "fetch": "https://developers.google.com/adwords/api/docs/terms",
       "select": ".devsite-article-body",
       "remove": "style"
-    }
+    },
+    "Ad Publishing Policy": {
```
Comment (clementbiron) [inline services/Google Ads.json:39]: Hey,

Ad Publishing Policy is not an available type of document. 
You should add it here : `OpenTermsArchive/blob/master/src/app/types.json`

Ask for any question you need 👌

File: services/Google Ads.json [hunk 2]
```diff
@@ -35,6 +35,10 @@
...
+    "fetch": "https://support.google.com/adsense/answer/9335564#",
+    "select": "div.article-content-container"
```
Comment (clementbiron) [inline services/Google Ads.json:41]: I think we can be more presice :

```
"select": ".article-container",
"remove": ".guide-button-container, .scClientFeedbackInlineFeedbacklinkroot"
```

**Review by clementbiron — CHANGES_REQUESTED** [review]

File: services/Google Ads.json (hunk 1 above)
Comment (AaronjSugarman) [reply services/Google Ads.json:39]: Done!

File: services/Google Ads.json (hunk 2 above)
Comment (AaronjSugarman) [reply services/Google Ads.json]: Okay, changed it!

Comment (clementbiron) [reply services/Google Ads.json]: ✔️

File: services/Google Ads.json (hunk 1 above)
Comment (clementbiron) [reply services/Google Ads.json:39]: As soon as [this PR](https://github.com/ambanum/OpenTermsArchive/pull/264) is corrected I can validate this one ;)

**Review by maxkey12 — APPROVED** [review]
gh pr checkout 262

**Pull request merged**